available/php/composer.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name":"Starters/Library",
+  "description":"Starter PHP project template",
+  "require": {
+    "php": "~7.3|~7.4|~8.0"
```

Comment:
I would have liked to require PHP 7.4 or 8 specifically, since 8 is the latest version and 7.4 was released a little over a year ago, but it seems the alpine package repository still only has 7.3